src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -3245,6 +3245,64 @@ void mvnw(Register Rd, Register Rm,
     f(pattern, 9, 5), f(0b0, 4), prf(pd, 0);
   }
 
+// SVE integer compare vector (wide-element variants are not included)
```

Comment:
The correct title for this section is "SVE Integer comparisons", as in the Arm ® Architecture Reference Manual Supplement, The Scalable Vector Extension". Please make sure all of these sections in the assembler have the same names as those in the manual.